docs/integrations/sources/shopify.md

```diff
@@ -5,77 +5,12 @@ description: >-
 
 # Shopify
 
-
 :::note
 
 Our Shopify Source Connector does not support OAuth at this time due to limitations outside of our control. If OAuth for Shopify is critical to your business, [please reach out to us](mailto:product@airbyte.io) to discuss how we may be able to partner on this effort.
 
 :::
 
-## Sync overview
-
-The Shopify source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
-
-This source can sync data for the [Shopify REST API](https://shopify.dev/api/admin-rest) and the [Shopify GraphQl API](https://shopify.dev/api/admin-graphql).
-
-## Troubleshooting
-
-Check out common troubleshooting issues for the Shopify source connector on our Discourse [here](https://discuss.airbyte.io/tags/c/connector/11/source-shopify).
-
-### Output schema
```

Comment:
why was this removed from here? Seems useful to have

Reply:
It still exists! It was just duplicated twice so I removed the first instance of it. It is now located above the Supported Streams but under the Setup Guide.